finat/point_set.py

```diff
@@ -34,15 +39,15 @@ def expression(self):
 
 
 class PointSingleton(AbstractPointSet):
-    """A point set representing a single point.
+    """A point set representing a single scalar point.
 
     These have a `gem.Literal` expression and no indices."""
 
     def __init__(self, point):
         """Build a PointSingleton from a single point.
 
-        :arg point: A single point which, once turned into a numpy array, has
-            shape (point dimension,)."""
+        :arg point: A single scalar point of shape (D,) where D is the
+            dimension of the point."""
```

Comment:
Same here with "scalar point", I would just drop the word scalar.

Reply:
Removed in 8333ae7bcca2fd963d3b952713d66be8b8026749